.github/workflows/ai-revision.yaml

```diff
@@ -0,0 +1,54 @@
+name: ai-revision
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to revise'
+        required: true
+        type: string
+        default: 'main'
+      file_names:
+        description: 'File names to revise'
+        required: false
+        type: string
+        default: ''
+      model:
+        description: 'Language model'
+        required: true
+        type: string
+        default: 'text-davinci-003'
+      branch_name:
+        description: 'Output branch'
+        required: true
+        type: string
+        default: 'ai-revision-davinci'
+
+jobs:
+  ai-revise:
+    name: AI Revise
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with: 
+          ref: ${{ inputs.branch }} 
+      - name: Install Manubot AI revision dependencies
+        run: pip install manubot[ai-rev]
```

Comment:
Hmm. Seems like there's a missing step to install python either via setup/python or via the conda env? Perhaps that's why the `aiohttp` error is occurring.

Reply:
I suggest taking a look at [miltondp/testing_rootstock_ai](https://github.com/miltondp/testing_rootstock_ai) instead, where I was trying some changes. I think we should try there and once it works, move the changes here. For instance, `pip install manubot[ai-rev]` doesn't work, because it downloads version 0.5.5 of manubot, which does not have the `ai-rev` extra. Or maybe we should publish a new manubot version in PyPI with this part?

Reply:
I still didn't have time to test a couple of changes, sorry. I will try to work on this by the end of the week.